devices/Mpu6886/Mpu6886.sln

```diff
@@ -0,0 +1,56 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30413.136
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{11A8DD76-328B-46DF-9F39-F559912D0360}") = "Mpu6886", "Mpu6886.nfproj", "{E79903D4-4F7C-4FB5-8078-086CD3B3252B}"
+EndProject
+Project("{11A8DD76-328B-46DF-9F39-F559912D0360}") = "Mpu6886.Samples", "samples\Mpu6886.Samples.nfproj", "{7EC388BC-55BB-4506-B93B-AD931F3F6D80}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Shared Projects", "Shared Projects", "{E785CAF1-60D9-4191-9AFB-C49027D3233A}"
+EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "BinaryPrimitives", "..\..\src\BinaryPrimitives\BinaryPrimitives.shproj", "{3F28B003-6318-4E21-A9B6-6C0DBD0BDBFD}"
+EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "System.Device.Model", "..\..\src\System.Device.Model\System.Device.Model.shproj", "{23325B14-3651-4879-9697-9846FF123FEB}"
+EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "System.Numerics", "..\..\src\System.Numerics\System.Numerics.shproj", "{F5CCB96C-70B1-4F0A-9F0E-BB912B6A996C}"
+EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Stopwatch", "..\..\src\Stopwatch\Stopwatch.shproj", "{875D133B-033B-4D11-B1A2-DEF898349AD5}"
+EndProject
+Global
+	GlobalSection(SharedMSBuildProjectFiles) = preSolution
+		..\..\src\System.Device.Model\System.Device.Model.projitems*{23325b14-3651-4879-9697-9846ff123feb}*SharedItemsImports = 13
+		..\..\src\BinaryPrimitives\BinaryPrimitives.projitems*{3f28b003-6318-4e21-a9b6-6c0dbd0bdbfd}*SharedItemsImports = 13
```

Comment:
we do have nugets for all those now. You can use the nuget tather than the shared project.

Reply:
Makes sense, so adjusting. 
@Ellerbach FYI [there are other device .sln files in the repo](https://github.com/nanoframework/nanoFramework.IoT.Device/search?q=23325b14-3651-4879-9697-9846ff123feb) that still use shared projects. Might need some cleanup later on.

Reply:
I know and they have to be adjusted over time. It's just a lot of work to do. While for the new ones, we can directly move to nuget.